charts/tidb-operator/values.yaml

```diff
@@ -185,30 +185,27 @@ admissionWebhook:
   ## if admissionWebhook.create and admissionWebhook.hooksEnabled.pods are both enabled,
   ## The pre-delete-job would delete the validationWebhookConfiguration using this image
   jobImage: "bitnami/kubectl:latest"
-  hooksEnabled:
+  ## validation webhook would check the given request for the specific resource and operation
+  validation:
     ## statefulsets hook would check requests for updating tidbcluster's statefulsets
     ## If enabled it, the statefulsets of tidbcluseter would update in partition by tidbcluster's annotation
     statefulSets: false
     ## pods hook would check requests for creating and deleting tidbcluster's pods
     ## if enabled it, the pods of tidbcluster would safely created or deleted by webhook instead of controller
     pods: true
     ## validating hook validates the correctness of the resources under pingcap.com group
-    validating: false
+    pingcapResources: false
+    ## mutation webhook would mutate the given request for the specific resource and operation
+  mutation:
     ## defaulting hook set default values for the the resources under pingcap.com group
-    defaulting: true
+    pingcapResources: true
   ## failurePolicy are applied to ValidatingWebhookConfiguration which affect tidb-admission-webhook
   ## refer to https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy
   failurePolicy:
-    ## deletePod Webhook would check the deleting request of tidbcluster pod and the failurePolicy is recommended as Fail
-    deletePod: Fail
-    ## createPod Webhook would check the creating request of tidbcluster pod and the failurePolicy is recommended as Ignore
-    createPod: Ignore
-    ## updateStatefulSet Webhook would check the updating request of tidbcluster statefulset and the failurePolicy is recommended as Ignore
-    updateStatefulSet: Ignore
-    ## validation hook validates the correctness of the resources under pingcap.com group
-    validating: Ignore
-    ## defaulting hook set default values for the the resources under pingcap.com group
-    defaulting: Ignore
+    ## the validation webhook would check the request of the given resources, and the failurePolicy is recommended as Fail
+    validation: Fail
+    ## the mutation webhook would mutate th erequest of the given resources, and the failurePolicy is recommended as Ignore
+    mutation: Ignore
```

Comment:
For the failurePlicy, I tend to use `validation` and `mutation` to cover all the admission webhooks in the coarse-grained level

Reply:
updated @DanielZhangQD